packages/@sanity/form-builder/src/inputs/BooleanInput.tsx

```diff
@@ -24,43 +23,43 @@ const BooleanInput = React.forwardRef(
       [onChange]
     )
 
+    const indeterminate = typeof value !== 'boolean'
+    const checked = value || false
+
+    const Input = layout === 'checkbox' ? Checkbox : Switch
+
     return (
       <ChangeIndicator>
-        <div className={styles.root} ref={ref}>
-          <div className={styles.inputWrapper} data-layout={layout}>
-            {layout === 'checkbox' && (
-              <Checkbox
-                ref={inputRef}
+        <Card as="label" border radius={1} padding={3}>
+          <Flex>
+            <Box style={{lineHeight: 0}}>
+              <Input
+                id={inputId}
+                ref={ref}
                 label={type.title}
-                readOnly={readOnly}
+                readOnly={Boolean(readOnly)}
                 onChange={handleChange}
                 onFocus={onFocus}
-                checked={value}
+                indeterminate={indeterminate}
+                checked={checked}
+                content={type.description}
```

Comment:
I'm unfamiliar with using the `content` attribute on input elements. What does it do?

Reply:
No idea what happened there. Removed.